blackberggroup/VA-iEX
# Feature/jls 6102292048/new iex stand alone repo

## What type of PR is this? (check all applicable)


## Description
From OHIL repo, create new stand-alone iEX repo.
Cleaned up repo: rm unused OHIL assets and coming-soon pages; edited deploy paths and exclusions; changed paths to use assets not assets-3-5; rm assets-3-5; menu dropdown arrow overrides.

## Related Tickets & Documents
- Monday Task #6102292048

## QA Instructions, Screenshots, Recordings
New iEX stand-alone repo. Compare to production. Review deploy scripts and confirm paths. 

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized
